### ISSUE TITLE
Can't exclude any folders on external storage 

### ISSUE THREAD

**Issue by Nogooduser** [OxygenCobalt/Auxio#150]
#### Describe the bug/crash:

At first I thought it wasn't implemented yet but I saw that you added support for exclusion on external storage devices in #134 so I think it's a bug that it's not working? 

Basically all the folders on my SD card can't be excluded. This is a problem because I have folders for audio books and I do not want them mixed with my music.

#### Steps To Reproduce the bug/crash:

1. Try to exclude any folder on an external storage device
2. App will say "this folder is not supported" 

#### Phone Information:

Samsung Galaxy A53 G running Android 12 (OneUI 4.1)

#### Due Diligence:
- [ X ] I have checked this issue for any duplicates.
- [ X ] I have checked for this issue in the [FAQ](https://github.com/OxygenCobalt/Auxio/blob/dev/info/FAQ.md).
- [ X ] I have read the [Contribution Guidelines](https://github.com/OxygenCobalt/Auxio/blob/dev/.github/CONTRIBUTING.md).

**Comment by OxygenCobalt** [comment]
Yep, this was a limitation I solved in f3a7813. Previously, I hard-coded the initial `/storage/emulated/0` directory when excluding directories, which resulted in external volume support being impossible. I've since reworked it to add support for external volumes, at least from Android Q onwards. On older versions, I can't trust that the folder representation of other volumes is stable across devices, so I do not implement it.

Feel free to try out this debug APK and let me know if it solves your problem. Some quick caveats:
- Due to (temporary) technical reasons, there is a 1-second delay between saving excluded directories and the app restarting.
- #128 is being worked on, so music loading may take longer than usual.
- While it works in a stock emulator (I don't have an SD card on a physical device), I have no idea how it works on weird OEM skins like OneUI.

[Auxio_External_Exclusion.zip](https://github.com/OxygenCobalt/Auxio/files/8849390/Auxio_External_Exclusion.zip)
